### PR TITLE
Discontinue 1.21 AMI's

### PR DESCRIPTION
**Pull request by cartermckinnon**
**Description of changes:**

Removes 1.21. Will include other associated logic related to the 1.21/1.22 divide.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by Issacwww — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -88,10 +88,6 @@ k8s: validate ## Build default K8s version of EKS Optimized AL2 AMI
 
 # Build dates and versions taken from https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 
-.PHONY: 1.21
```
Comment (ravisinha0506) [inline Makefile:91]: Can we also remove 1.21 from line# 40?

Comment (Issacwww) [reply Makefile:91]: good catch, I believe we should remove it

**Review by Issacwww — CHANGES_REQUESTED** [review]
revoke approval: please make the change to remove 1.21 from line#40

**Comment by cartermckinnon** [comment]
@Issacwww the team discussed the `all` target, which we decided isn't the right default target for this Makefile. I've replaced it here with a `latest` target; take a look when you get a chance!

**Review by Issacwww — APPROVED** [review]

**Pull request merged**